sycl/include/CL/sycl/image.hpp

```diff
@@ -50,9 +50,9 @@ enum class image_channel_type : unsigned int {
   signed_int32 = 9,
   unsigned_int8 = 10,
   unsigned_int16 = 11,
-  unsigned_int32 = 13,
-  fp16 = 14,
-  fp32 = 15
+  unsigned_int32 = 12,
+  fp16 = 13,
+  fp32 = 14
```

Comment:
I'm afraid, this will break ABI

Reply:
I believe the ABI was broken with the previous change, this restores it: https://github.com/intel/llvm/commit/6cb162169bc68e1c17f83ae67cb60f311bb4afc6#diff-558e56b2d3dda7f222ba5f1dce9c3dc7afbe514f431828eb2de090cba91c463bR52-R53

(hmm - that link isn't passing through my selection, look at the change between line 52-53 )